BoardTests.java

```diff
@@ -56,11 +69,21 @@ public void checkForDraw() {
 
     @Test
     public void canGetAllAvailablePositions() {
-        Board board = new Board("---x----");
-        int count = 0;
-        for (int i=0; i < board.availablePositions().size(); i++) {
-            count += 1;
+        Board board = new Board("---x-----");
+        ArrayList<Integer> positions = board.availablePositions();
+        for (int i=0; i < positions.size(); i++) {
+            int position = positions.get(i);
+            assertThat(board.availablePosition(position - 1), is(true));
```

Comment:
I don't like this test, it is over complicated because of the for loop.
I would rather just assert that availablePositions returns an ArrayList which matches the expected values.. (can't remember if there are any complications with comparing ArrayLists in Java, but there will be a way to get around it..)